src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/ProductController.php

```diff
@@ -337,7 +338,24 @@ private function formatProductsForAssociation(array $productsForAssociation): ar
      *
      * @return Response
      */
-    private function renderProductForm(FormInterface $productForm, ?int $productId = null): Response
+    private function renderCreateProductForm(FormInterface $productForm, ?int $productId = null): Response
```

Comment:
I don't understand why you have a parameter `productId` on rendering the creation form?
I think you have to remove this parameter

Reply:
You're right it's an old copy/paste I need to clean it